java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -407,4 +407,19 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<Long> objectRefcount(String key);
+
+    /**
+     * Updates the last access time of specified <code>keys</code> and returns the count of keys that
+     * were updated.
+     *
```

Comment:
Add a multi key note https://github.com/Bit-Quill/glide-for-redis/pull/226/files#diff-46c31f91e6bb4a1d2c763cddafd17d4aaad1b24f91744b604d6f11a26da9d45dR21-R22

Reply:
It does not need a multi key note! Works fine with different slots.